date.go

```diff
@@ -124,11 +124,11 @@ func (d *dateObject) setTimeMs(ms int64) Value {
 }
```

Comment:
Do we need to set `d.invalid` to `false` on line 118 within `setTimeMs` in case we're going from an invalid value to a valid one?